hail/python/hail/plot/plots.py

```diff
@@ -360,10 +360,18 @@ def cumulative_histogram(data, range=None, bins=50, legend=None, title=None, nor
            range=nullable(sized_tupleof(nullable(sized_tupleof(numeric, numeric)),
                                         nullable(sized_tupleof(numeric, numeric)))),
            title=nullable(str), width=int, height=int,
-           font_size=str, colors=sequenceof(str))
-def histogram2d(x, y, bins=40, range=None,
-                 title=None, width=600, height=600, font_size='7pt',
-                 colors=bokeh.palettes.all_palettes['Blues'][7][::-1]):
+           font_size=str, colors=sequenceof(str),
+           log=bool)
+def histogram2d(x: hl.expr.NumericExpression,
+                y: hl.expr.NumericExpression,
+                bins: int = 40,
+                range: Tuple[int, int] = None,
+                title: str = None,
+                width: int = 600,
+                height: int = 600,
+                font_size: str = '7pt',
+                colors: List[str] = bokeh.palettes.all_palettes['Blues'][7][::-1],
+                log: bool = False):
```

Comment:
I would generally prefer this, but I believe Hail house style has been to not use typehints in function signatures, since it can mess with linters, but to leave it to the typechecker above. @tpoterba confirm?

Reply:
Can revert if desired but my other code in plots has typehints 😛

Reply:
happy to use type hints - just not for expressions with type polymorphism where  you'd really need to use type generics (Array[T].map => Array[U])